roles/openshift_master/defaults/main.yml

```diff
@@ -29,4 +29,4 @@ os_firewall_deny:
 - service: former etcd peer port
   port: 7001/tcp
 
-openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
I think these defaults actually need to be nested and not chained..   for example if openshift_image_tag is not defined, I would expect this to throw an error and not use the value of the next default filter.